### PR TITLE
Update css with new colours

### PR DESCRIPTION
**Pull request by osahyoun**
Updating CSS with new brand colours

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204864989115518

image
image
image
image

### CODE REVIEW

File: source/javascripts/sumofus/backbone/campaigns.js [hunk 1]
```diff
@@ -25,7 +25,9 @@ const Campaigns = Backbone.View.extend({
   },
 
   success(data) {
+    console.log('see data here!!!!', data, data.length, this.limit);
```
Comment (yeseniamolinab) [inline source/javascripts/sumofus/backbone/campaigns.js:28]:
```suggestion
```

File: source/javascripts/sumofus/backbone/campaigns.js [hunk 2]
```diff
@@ -78,9 +81,9 @@ const Campaigns = Backbone.View.extend({
     donationPage
   ) {
     var showImage = 'visibility: visible';
-    if (allImagesUrl.length === 0) {
-      showImage = 'visibility: hidden';
-    }
+    // if (allImagesUrl.length === 0) {
```
Comment (yeseniamolinab) [inline source/javascripts/sumofus/backbone/campaigns.js:84]: @osahyoun, any reason to leave these lines commented? can you delete them instead?

File: Gemfile.lock [hunk 3]
```diff
@@ -158,4 +158,4 @@ RUBY VERSION
    ruby 2.4.1p111
 
 BUNDLED WITH
-   1.17.1
+   2.1.4
```
Comment (yeseniamolinab) [inline Gemfile.lock:161]:
```suggestion
   1.17.1
```

If you revert this, the build will pass 👍🏼

**Review by yeseniamolinab — CHANGES_REQUESTED** [review]
Hey @osahyoun, thanks for handling this!
It is looking good, and I left a few minor suggestions. Could you let me know if those make sense to you?

File: source/javascripts/sumofus/backbone/campaigns.js (hunk 2 above)
Comment (osahyoun) [reply source/javascripts/sumofus/backbone/campaigns.js:84]: Thanks, @yeseniamolinab, for the catch

File: Gemfile.lock (hunk 3 above)
Comment (osahyoun) [reply Gemfile.lock:161]: 💯

**Review by yeseniamolinab — APPROVED** [review]

**Pull request merged**